lib/backends/sdc/build.js

```diff
@@ -891,8 +894,266 @@ function runBuildCommand(opts, callback) {
+        });
+
+        commitEventStream = new LineStream();
+        commitSocket.pipe(commitEventStream);
+        commitEventStream.on('line', function _commitLineCb(event) {
+            log.debug('commit: got commit event:', String(event));
+            try {
+                event = JSON.parse(event);
+            } catch (e) {
+                log.error('Commit: invalid json: %s - ignoring', event);
+                return;
+            }
+            switch (event.type) {
+                case 'end':
+                    if (event.error) {
+                        // Will be passed to callback `cb`.
+                        commitError = new Error(event.error);
+                    }
+                    break;
+                case 'message':
+                case 'stdout':
+                    log.info('Commit event message: %s', event.message);
+                    break;
+                case 'image_create':
+                    createImage(event.payload, { rat: rat, req: req },
+                        function _imageCreateCb(err, result)
+                    {
+                        sendEventResponse(event, err, result);
+                    });
+                    break;
+                case 'commit_finished':
+                    finalImageId = event.finalId;
+                    vasync.pipeline({ funcs: [
+                        function _doAddBaseImageHead(_result, next) {
+                            addBaseImageHead(baseImageId, finalImageId,
+                            {
+                                baseImageIndexName: baseImg.index_name,
+                                req: req,
+                                scratchImage: scratchImage
+                            }, next);
+                        },
+                        function _doTagImage(_result, next) {
+                            tagImage({
+                                    docker_id: finalImageId,
+                                    name: opts.tag,
+                                    req: req
+                                }, next);
+                        }
+                    ]}, function commitFinishPipeCb(err) {
+                        if (!err) {
+                            commitFinishedSuccessfully = true;
+                        }
+                        sendEventResponse(event, err);
+                    });
+                    break;
+                default:
+                    log.error('Unhandled commit event: %j', event);
+                    break;
+            }
+        });
+        commitEventStream.on('close', function _eventStreamCloseCb() {
+            log.debug('commit: commitEventStream.close - closing socket');
+        });
+        commitSocket.on('end', function _commitSocketEndCb() {
+            log.debug('commit: commitSocket.end', commitError);
+            cb(commitError);
+        });
+
+        log.debug('commit: creating second connection to cnapi');
```

Comment:
Shouldn't that log message be outputted close to the call to `net.createConnection` at the beginning of the function so that it's clearer what it refers to?